lib/_init.js

```diff
@@ -3,9 +3,28 @@
 const taiko = require("taiko");
 const path = require("path");
 
-beforeSuite(async () => {
-  await taiko.openBrowser({ headless: true });
-});
+/**
+ *  Get headless option from env file
+ */
+const headless = process.env.headless_chrome.toLowerCase() === "true";
+
+/**
+ *  Get browser arguments form env file
+ */
+const browser_args = process.env.browser_args;
+
+/**
+ *  If there are no headless option and browser arguments are given in the env file, taiko will open browser with default values.
+ */
+if (browser_args && headless) {
```

Comment:
Assume that both of these are optional and could be used independently. For example, use headless option without any additional browser arguments.

Reply:
Also, this will execute line 21 only when headless is true.